src/commands/run.ts

```diff
@@ -57,6 +57,11 @@ export class RunCommand extends ApifyCommand<typeof RunCommand> {
             description: 'Deletes all records from the default key-value store in the local directory before the run starts, except for the "INPUT" key.',
             required: false,
         }),
+        'npm-script': Flags.string({
```

Comment:
What about calling this `entrypoint` or something else instead? Just a thought, I don't like the word `npm` here very much. Or maybe `run-script`?

Reply:
Changing the name will be finicky because we'd also have to do it properly for python (and sadly i have to python knowledge about how that would work)

Reply:
But sure, I'll rename it

Reply:
isn't it just about doing the same with `executableLocation` in that python branch?

Reply:
cc @vdusek

Reply:
I discussed it with @vladfrangu on Slack, I believe it can be marked as resolved.

Python (Pip) does not support anything like Node.js `package.json` scripts.

So it will have to be just a path to some Python executable I suppose.

Reply:
> Python (Pip) does not support anything like Node.js package.json scripts.

But we run python from a path, by default its `src`, to me this feels exactly the same - to be able to redefine the default `src` path. Its not relevant that there are no "npm scripts", you can execute stuff in different locations in any language.